pkg/tnf/handlers/handler_template/template.tmpl

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2020-2021 Red Hat, Inc.
```

Comment:
the name of the file should be updated. Is this the test template ? or the handler template ?